src/sentry/api/endpoints/organization_metrics.py

```diff
@@ -246,25 +243,45 @@ class OrganizationMetricsTagDetailsEndpoint(OrganizationEndpoint):
 
     def get(self, request: Request, organization, tag_name) -> Response:
         metric_names = request.GET.getlist("metric") or []
+        if len(metric_names) > 1:
+            raise ParseError(
+                "Please supply only a single metric name. Specifying multiple metric names is not supported for this endpoint."
+            )
```

Comment:
we should aim to move towards a `metrics/meta/<mri>/` for single metrics info so this is a good middle ground for now!